spec/spec_helper.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.